### DISCUSSION TITLE
Screen coordinates of origin

### DISCUSSION THREAD

**Discussion by Mielie** [epezent/implot#317] [Q&A]
Hi,

Just started using ImPlot and would like to draw some additional graphics onto the plot and was wondering if its possible to easily get the screen coordinates for the origin point of the axis?

**Comment by epezent** [comment]
GetPlotPos / GetPlotSize is what you want. Also PlotToPixels if you need to convert coordinates to screen position.

**Reply by Mielie** [reply]
Thats fantastic thanks, just what I needed! Whats the correct process for drawing over a graph? If I get the drawlist and add to it, this appears underneath the graph when rendered?

**Reply by epezent** [reply]
Yes, use GetPlotDrawList between Begin/End plot. You can set the axis grid to render in the background (ie in BeginPlot) or foreground (ie in EndPlot) by setting the correct ImPlotAxisFlag to SetupAxis. 

You can also cache the coordinates and render after EndPlot later on in the frame. In that case it will be on top of everything. 

Checkout Custom Rendering in implot_demo.cpp

**Reply by Mielie** [reply]
Thanks for the very helpful replies.
